.github/workflows/release-rancher.yaml

```diff
@@ -59,6 +62,36 @@ jobs:
           # Allow making git push request later on
           persist-credentials: true
 
+      - name: Find charts branch
+        id: find_charts_branch
+        run: |
+          cd rancher
+          # Extract dev-v2.9 out of the following line:
+          #     ChartDefaultBranch                  = NewSetting("chart-default-branch", "dev-v2.9")
```

Comment:
Should this be `v2.11`?

Reply:
It's just an example comment and I think for some of these PRs I changed them and for some I didn't. I figured I wouldn't want to change those comments all the time we create new branches

Reply:
Ope yep you're right, got confused by the `ChartDefaultBranch = NewSetting("chart-default-branch", "dev-v2.9")` below it, not even the same language. LGTM